README.md

```diff
@@ -1,2 +1,372 @@
+
+There are various options to choose from here, we will choose the `Container Image` where you will use the existing image you previously built and pushed to DockerHub and deploy it on your OpenShift Container Platform.
+
+![OpenShift web console create app with container image](doc/images/OpenShift-GUI-dev-perspective-app-creation-choices-tile-highlighted-container-image.png)
+
+<details><summary><strong>Learn more about application creation options as an OpenShift Developer</strong></summary>
+
+
+1. From Git: Use this option to import an existing codebase in a Git repository to create, build, and deploy an application on OpenShift Container Platform.
+
+1. Container Image: Use existing images from an image stream or registry to deploy it on to OpenShift Container Platform.
+
+1. From Catalog: Explore the Developer Catalog to select the required applications, services, or source to image builders and add it to your project.
+
+1. From Dockerfile: Import a dockerfile from your Git repository to build and deploy an application.
+
+1. YAML: Use the editor to add YAML or JSON definitions to create and modify resources.
+
+To learn more check out the [OpenShift developer documentation](https://docs.openshift.com/container-platform/4.3/applications/application_life_cycle_management/odc-creating-applications-using-developer-perspective.html#odc-creating-applications-using-developer-perspective)
+</details>
+
+Fill out the image name text box on your DockerHub repository.  It should be:
+
+`$DOCKERHUB_USERNAME/currencyexchange-py:v0.0.1`
+
+where `$DOCKERHUB_USERNAME` is you Docker hub username.
+
+Click on the `search icon`  (magnifying glass) to the left of the text box.
+
+It should fetch the image metadata from DockerHub and create a nice form.
+
+![OpenShift Container image form](doc/images/OpenShift-container-image-after-search-details-of-image.png)
+
+Click on the blue create button indicated with the number one(1) above.
+
+Great!  You should now see a nice topology and summary of the application you just created:
+
+![OpenShift](doc/images/OpenShift-GUI-container-image-created-successfully-maybe.png)
+
+Click on the `currencyexchang...` button in the above screenshot, indicated by the number one(1) in the orange triangle.
+
+You should now be able to click on the public URL  under the `Routes` section in the right hand panel. 
+
+For example:
+
+`http://currencyexchange-py-currencyexchange-py.username-ubi-webconso-f2c6cdc6801be85b09d0xxxxx06f13e3-0000.us-northeast.containers.appdomain.cloud/`
+
+You should now see Python Flask swagger API interactive web page, where you can manually test the Currency Conversion as seen below. 
+
+![OpenShift url shows swagger and exchange rate conversion executes as expected](doc/images/OpenShift-url-navigate-to-shows-swagger-success.png)
+
+
+
+### Congratulations!  
+
+You have now successfully created an microservice using the `RedHat Universal Base Image (UBI)` on `OpenShift 4`
```

Comment:
You have now successfully created and deployed a microservice using the RedHat Universal Base Image (UBI) in an OpenShift 4 cluster on IBM Cloud